common.js

```diff
@@ -1,8 +1,9 @@
 // @flow
 
+import type { ElementProps } from 'react';
 import { requireNativeComponent, View } from 'react-native';
 
-export default requireNativeComponent('BVLinearGradient', null);
```

Comment:
Just to be clear this second argument has been removed in RN?

Reply:
Yep! Here's the error it surfaced:

![](https://user-images.githubusercontent.com/73652/54565891-52eb9080-49c7-11e9-8be2-c24fde1634b8.png)

Interestingly enough, the [example on the official docs](https://facebook.github.io/react-native/docs/native-components-ios) is also wrong 🙈 (I should make a PR to fix that too).

Reply:
https://github.com/facebook/react-native-website/pull/840

Reply:
You're on fire 🔥